updatecli/updatecli.d/docker-images/nginx.yaml

```diff
@@ -11,38 +11,35 @@ scms:
       token: "{{ requiredEnv .github.token }}"
       username: "{{ .github.username }}"
       branch: "{{ .github.branch }}"
+  nginx-gh-mirror:
+    kind: git
+    spec:
+      url: "https://github.com/nginx/nginx.git"
```

Comment:
The `.git` is mandatory?

Reply:
Haven't tried without but not sure why it would be a problem ?

Reply:
Not a problem per se, just less stuff, shorter, more direct (as I asked myself if it was mandatory)

Reply:
(seems to work: removed it)